spec/serializers/enrichment_job_serializer_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe EnrichmentJobSerializer do
+  let!(:enrichment_job) { FactoryBot.create(:enrichment_job) }
```

Comment:
You can configure `FactoryBot` to not need the class before the create. So it would be just 

```ruby
let!(:enrichment_job) { create(:enrichment_job) }
```